test/e2e/volume_provisioning.go

```diff
@@ -122,25 +122,9 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			testDynamicProvisioning(c, claim)
 		})
 	})
-
-	framework.KubeDescribe("DynamicProvisioner Alpha", func() {
```

Comment:
Instead of removing the alpha test completely, can we modify this test to test the default storageclass behavior on these providers?